scipy/stats/_continuous_distns.py

```diff
@@ -8605,6 +8605,102 @@ def _rvs_scalar(self, a, b, numsamples=None, random_state=None):
 truncnorm = truncnorm_gen(name='truncnorm', momtype=1)
 
 
+class truncpareto_gen(rv_continuous):
+    r"""An upper truncated Pareto continuous random variable.
+
+    %(before_notes)s
+
+    See Also
+    --------
+    pareto : Pareto distribution
+
+    Notes
+    -----
+    The probability density function for `truncpareto` is:
+
+    .. math::
+
+        f(x, b, c) = \frac{b}{1 - c^{-b}} \frac{1}{x^{b+1}}
+
+    for :math:`b > 0`, :math:`c > 1` and :math:`1 \le x \le c`.
+
+    `truncpareto` takes `b` and `c` as shape parameters for :math:`b` and
+    :math:`c`.
+
+    Notice that the upper truncation value :math:`c` is defined in
+    standardized form so that random values of an unscaled, unshifted variable
+    are within the range ``[1, c]``.
+    If `u_r` is the upper bound to a scaled and/or shifted variable,
+    then `c = (u_r - loc) / scale`. In other words, the support of the
+    distribution becomes `(scale + loc) <= x <= (c*scale + loc)` when
+    `scale` and/or `loc` are provided.
```

Comment:
Here's how this is rendering:
![image](https://user-images.githubusercontent.com/6570539/176800984-a0e010b5-ce6f-429b-afce-5350eabd61c5.png)

According to the numpydoc style guide, single backticks are appropriate for "variables". If we want code to render in monospaced font, though, double backticks are needed. 

These rules are really ambiguous, but here is how I normally would apply them.
```suggestion
    Notice that the upper truncation value :math:`c` is defined in
    standardized form so that random values of an unscaled, unshifted variable
    are within the range ``[1, c]``.
    If ``u_r`` is the upper bound to a scaled and/or shifted variable,
    then ``c = (u_r - loc) / scale``. In other words, the support of the
    distribution becomes ``(scale + loc) <= x <= (c*scale + loc)`` when
    `scale` and/or `loc` are provided.
```

As parameters of the distribution, `scale` and `loc` are definitely "variables", so I've left the single backticks alone. (Why they shouldn't always render in monospaced font, and why it is correct for them to render in bold
![image](https://user-images.githubusercontent.com/6570539/176801946-58ff8e62-f28a-4345-8d83-3eaeb7359a21.png)
italics,
![image](https://user-images.githubusercontent.com/6570539/176802018-c0bdcc61-8037-466b-a9e1-abf856bc9488.png)
or monospaced font
![image](https://user-images.githubusercontent.com/6570539/176802097-9abc17d5-373b-4acd-8425-3c1a15dfb153.png)
depending on context is beyond me, so don't worry that this was not intuitive.)

Because `c = (u_r - loc) / scale` and `(scale + loc) <= x <= (c*scale + loc)` should render in monospaced font, I've given them double backticks.

`u_r` is questionable. I suppose it's a "variable"? But it's not actually a parameter or even a variable in the code, so since I would prefer it to render in monospaced font as it does in `c = (u_r - loc) / scale`, I've given it double backticks. That's a judgment call.